### PR TITLE
Fix for the warning "Method override for the designated initializer of the superclass '-init' not found"

### PR DESCRIPTION
**Pull request by ksuther**
Fixes a new warning on Xcode 7 about designated initializers.

### CODE REVIEW

**Comment by rivera-ernesto** [comment]
I think this should be pull requested to the `swift_2.0` branch.

**Comment by ksuther** [comment]
I can do that, but what's the reason? I'm running off of master and see the warning there.

File: Classes/DDFileLogger.m [hunk 1]
```diff
@@ -1122,6 +1122,10 @@ + (instancetype)logFileWithPath:(NSString *)aFilePath {
     return [[self alloc] initWithFilePath:aFilePath];
 }
 
+- (instancetype)init {
+    @throw nil;
```
Comment (rivera-ernesto) [inline Classes/DDFileLogger.m]: Because this is not supported in swift 1.2.

**Comment by ksuther** [comment]
It looks like the header part of this was already added to the swift_2.0 branch in commit 58ae948213e76a336473825256055b75d8da30b3. I'll close this. I assume the swift_2.0 branch will be getting merged into master relatively soon? I'm not actually using Swift at all, I just don't want these extraneous warnings appearing when I build :)

**Pull request closed without merging**